gql2sql_wasm/Cargo.toml

```diff
@@ -10,17 +11,8 @@ crate-type = ["cdylib", "rlib"]
 console_error_panic_hook = { version = "0.1.7", optional = true }
 gql2sql = { path = "../gql2sql" }
 graphql-parser = "0.4.0"
-js-sys = "0.3.61"
 serde = { version = "1.0", features = ["derive"] }
-serde-wasm-bindgen = "0.4"
 wasm-bindgen = "0.2.84"
-wee_alloc = "0.4.5"
-
-[dependencies.web-sys]
-version = "0.3"
-features = [
-  "console",
-]
```

Comment:
I'm not 100% sure but I *think* we need these.

Reply:
I can add those back in

I ran `cargo +nightly udeps` and it said they weren't being used. I ran `cargo check` and it compiled. 

But I can add those back since they don't hurt anything.

Reply:
I keep getting alerts about `wee_alloc`

https://www.reddit.com/r/rust/comments/x1cle0/dont_use_wee_alloc_in_production_code_targeting/

![Screenshot 2023-02-27 at 15 23 17](https://user-images.githubusercontent.com/38776747/221699551-5c18bf0b-cd06-4bc2-b7b7-1cf3338df33f.png)

Reply:
I can add it back for the time being though if you would like

Reply:
Interesting. Ok, we can leave it out.